templating/twig_extension.rst

```diff
@@ -58,14 +58,114 @@ Create a class that extends ``\Twig_Extension`` and fill in the logic::
     `global variables`_.
 
 Register an Extension as a Service
-----------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Next, register your class as a service and tag it with ``twig.extension``. If you're
 using the :ref:`default services.yml configuration <service-container-services-load-example>`,
 you're done! Symfony will automatically know about your new service and add the tag.
 
 You can now start using your filter in any Twig template.
 
+Creating Lazy-Loaded Twig Extensions
+------------------------------------
+
+.. versionadded:: 1.26
```

Comment:
Don't we refer to Symfony versions here? Having a Twig version seems weird to me.

Reply:
Yes, it's not very common, but we add those version numbers for Twig and other apps different than Symfony in some places of the docs.